xml/SLES-JeOS-12SP4-Quickstart.xml

```diff
@@ -431,6 +409,13 @@
   <sect2 xml:id="sec-openstack-cloud">
    <title>OpenStack Cloud</title>
 
+   <para>This is our image to be used in a OpenStack Cloud environment</para>
+   <important>
+   <title>cloud-init is replacing jeos-firstboot</title>
```

Comment:
That is misleading - jeos-firstboot is not and never was on the OpenStack flavor, so it should be `cloud-init is used in place of jeos-firstboot`